.travis.yml

```diff
@@ -72,13 +72,6 @@ jobs:
         FILE_ENV="./ci/test/00_setup_env_arm.sh"
         QEMU_USER_CMD=""
 
-    - stage: test
-      name: 'S390x  [GOAL: install]  [buster]  [unit tests, functional tests]'
-      arch: s390x  # Can disable QEMU_USER_CMD and run the tests natively without qemu
-      env: >-
-        FILE_ENV="./ci/test/00_setup_env_s390x.sh"
```

Comment:
this file is removing too?

Reply:
I'd like to keep it so that it can be executed easier, if needed